index.html

```diff
@@ -1,19 +1,89 @@
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <img src="img/karma-logo.svg" alt="logo" width="40" height="40" />
+        <img
+          class="menu-burger"
+          src="img/menu-hamburger.svg"
+          alt="menu-burger"
+        />
+        <ul>
+          <li>Meet Karma</li>
+          <li>How it Works</li>
+          <li>Store</li>
+          <li>Blog</li>
+          <li>Help</li>
+          <li>Login</li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="first-section" title="Background a person using a device">
+        <div class="content">
+          <h1 class="content-title first">Introducing Karma</h1>
+          <p class="content-paragraph">
+            Bring WiFi with you, everywhere you go.
+          </p>
+          <button>Learn More</button>
+        </div>
+      </section>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+      <section class="second-section">
+        <h1 class="content-title">Everyone needs a little Karma.</h1>
+        <div class="product">
+          <div class="prod">
+            <img src="img/icon-devices.svg" alt="devices" height="100px" />
+            <p class="prod-paragraph">Internet for all devices</p>
+          </div>
+          <div class="prod">
+            <img src="img/icon-coffee.svg" alt="coffee" height="100px" />
+            <p class="prod-paragraph">Internet for all devices</p>
+          </div>
+          <div class="prod">
+            <img src="img/icon-refill.svg" alt="refill" height="100px" />
+            <p class="prod-paragraph">Internet for all devices</p>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <h1 class="footer-title">Join us on</h1>
+      <div class="social-media">
+        <div class="twitter">
+          <img src="img/twitter-icon.svg" alt="twitter-icon" height="20px" />
+        </div>
 
-</body>
+        <div class="facebook">
+          <img src="img/facebook-icon.svg" alt="facebook-icon" height="20px" />
+        </div>
+
+        <div class="instagram">
+          <img
+            src="img/instagram-icon.svg"
+            alt="instagram-icon"
+            height="20px"
```

Comment:
Consider reuse of a same CSS class on these 3 icon DIVs? It would be nice if you can keep the same code style for them as well.